apis/r/R/write_soma.R

```diff
@@ -466,7 +466,8 @@ write_soma.TsparseMatrix <- function(
     shape = shape %||% dim(x),
     ingest_mode = ingest_mode,
     platform_config = platform_config,
-    tiledbsoma_ctx = tiledbsoma_ctx
+    tiledbsoma_ctx = tiledbsoma_ctx,
+    tiledb_timestamp = Sys.time()
```

Comment:
NULL may be better.  Let's core / libtiledbsoma do this.

Reply:
`NULL` wasn't working, setting to `Sys.time()` got the tests to pass